cluster-provision/gocli/opts/istio/manifests/istio-operator-with-cnao.cr.yaml

```diff
@@ -18,6 +18,7 @@ spec:
     global:
       jwtPolicy: third-party-jwt
     cni:
+      provider: multus
```

Comment:
Why is it necessary to add this option? 
Note that we specify "provider: multus" in line 32.